### PR TITLE
Updating steps in link-visio-drawing-to-a-specific-region.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Support/issues/1329.

### CODE REVIEW

**Comment by dariomws** [comment]
Closes https://github.com/MicrosoftDocs/OfficeDocs-Support/issues/1349.

File: Office/Client/excel/link-visio-drawing-to-a-specific-region.md [hunk 1]
```diff
@@ -41,8 +41,8 @@ To embed and link a named portion of an Excel worksheet into a Visio drawing, fo
 
 1. Open your worksheet in Microsoft Excel.
 2. Select the range of cells that you want to link to.
-3. On the Insert menu, point to Name, and then click Define.
-4. In the Define Name dialog box, type a unique name in the **Names in Workbook** box, and then click OK.
+3. On the Formulas menu, click Define Name.
```
Comment (JohanFreelancer9) [inline Office/Client/excel/link-visio-drawing-to-a-specific-region.md:44]:
```suggestion
3. On the Formulas menu, click **Define Name**.
```

File: Office/Client/excel/link-visio-drawing-to-a-specific-region.md [hunk 2]
```diff
@@ -41,8 +41,8 @@ To embed and link a named portion of an Excel worksheet into a Visio drawing, fo
...
+4. In the Define Name dialog box, type a unique name and then click OK.
```
Comment (JohanFreelancer9) [inline Office/Client/excel/link-visio-drawing-to-a-specific-region.md:45]:
```suggestion
4. In the Define Name dialog box, type a unique name and then click **OK**.
```

File: Office/Client/excel/link-visio-drawing-to-a-specific-region.md [hunk 3]
```diff
@@ -41,8 +41,8 @@ To embed and link a named portion of an Excel worksheet into a Visio drawing, fo
...
 5. Save the Microsoft Excel Workbook.
 6. Start Microsoft Visio and open your drawing.
 7. On the Insert menu, click Object.
```
Comment (JohanFreelancer9) [inline Office/Client/excel/link-visio-drawing-to-a-specific-region.md:48]:
```suggestion
7. On the Insert menu, click **Object**.
```

File: Office/Client/excel/link-visio-drawing-to-a-specific-region.md [hunk 4]
```diff
@@ -52,10 +52,10 @@ To embed and link a named portion of an Excel worksheet into a Visio drawing, fo
    > [!NOTE]
    > Selecting the **Link to file** check box establishes a dynamic connection between the Visio drawing and another program's file. When changes occur in the original file, you can update the linked file so that the most current version of the object appears in the linked file.
 
-10. Select the Excel worksheet. On the Edit menu, click Links.
-11. In the Links dialog box, click Automatic under Update, and then click Change Source.
+10. In the **Tell me what you want to do** box, type and click Links.
```
Comment (JohanFreelancer9) [inline Office/Client/excel/link-visio-drawing-to-a-specific-region.md:55]:
```suggestion
10. In the **Tell me what you want to do** box, type and click Links.
```

File: Office/Client/excel/link-visio-drawing-to-a-specific-region.md [hunk 5]
```diff
@@ -52,10 +52,10 @@ To embed and link a named portion of an Excel worksheet into a Visio drawing, fo
...
+11. In the Links dialog box, select the link to the Excel worksheet, click **Automatic** under Update, and then click **Change Source**.
 12. In the Item Name box, type the name defined in step 4, and then click Open.
```
Comment (JohanFreelancer9) [inline Office/Client/excel/link-visio-drawing-to-a-specific-region.md:57]:
```suggestion
12. In the Item Name box, type the name defined in step 4, and then click **Open**.
```

File: Office/Client/excel/link-visio-drawing-to-a-specific-region.md [hunk 6]
```diff
@@ -52,10 +52,10 @@ To embed and link a named portion of an Excel worksheet into a Visio drawing, fo
...
-13. Click Update Now, and then click Close.
+13. Click **Update Now**, and then click Close.
```
Comment (JohanFreelancer9) [inline Office/Client/excel/link-visio-drawing-to-a-specific-region.md:58]:
```suggestion
13. Click **Update Now**, and then click **Close**.
```

File: Office/Client/excel/link-visio-drawing-to-a-specific-region.md [hunk 7]
```diff
@@ -52,10 +52,10 @@ To embed and link a named portion of an Excel worksheet into a Visio drawing, fo
    > [!NOTE]
    > Selecting the **Link to file** check box establishes a dynamic connection between the Visio drawing and another program's file. When changes occur in the original file, you can update the linked file so that the most current version of the object appears in the linked file.
 
-10. Select the Excel worksheet. On the Edit menu, click Links.
-11. In the Links dialog box, click Automatic under Update, and then click Change Source.
+10. In the **Tell me what you want to do** box, type and click Links.
+11. In the Links dialog box, select the link to the Excel worksheet, click **Automatic** under Update, and then click **Change Source**.
```
Comment (JohanFreelancer9) [inline Office/Client/excel/link-visio-drawing-to-a-specific-region.md:56]:
```suggestion
11. In the Links dialog box, select the link to the Excel worksheet, choose **Automatic** under Update, and then select **Change Source**.
```

File: Office/Client/excel/link-visio-drawing-to-a-specific-region.md (hunk 3 above)
Comment (JohanFreelancer9) [inline Office/Client/excel/link-visio-drawing-to-a-specific-region.md:48]:
```suggestion
7. On the Insert menu, click **Object**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**